tests/snmp/test_snmp_memory.py

```diff
@@ -54,13 +54,11 @@ def test_snmp_memory(duthosts, enum_rand_one_per_hwsku_hostname, localhost, cred
                                 community=creds_all_duts[duthost]["snmp_rocommunity"], wait=True)['ansible_facts']
     facts = collect_memory(duthost)
     compare = (('ansible_sysTotalFreeMemery', 'MemFree'), ('ansible_sysTotalBuffMemory', 'Buffers'),
-               ('ansible_sysCachedMemory', 'Cached'))
+               ('ansible_sysCachedMemory', 'Cached'), ('ansible_sysTotalSharedMemory', 'Shmem'))
```

Comment:
Why are we adding shared memory here? Then this will be part of the check in L64 where we will check the snmp data for this field with the system data is within 4%. The description of the PR mentions that 4% might not be accurate. Am I missing something here?